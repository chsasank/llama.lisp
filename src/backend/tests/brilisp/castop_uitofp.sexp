(brilisp
    (define ((fprint float) (n float)))
    (define ((dprint double) (n double)))

    (define ((main void))
        (set (a int32) (const 257))
        (set (b float) (uitofp a float))
        (set (tmp float) (call fprint b))

        (set (a int8) (const -1))
        (set (b double) (uitofp a double))
        (set (tmp double) (call dprint b))
        (ret)))
