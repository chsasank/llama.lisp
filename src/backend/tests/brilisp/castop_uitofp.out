257.000000
255.000000
